Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN apt-get update \
 
 RUN git clone https://github.com/nevermined-io/rapidsnark \
```

Comment:
Maybe you can do:
`git clone --branch <branchname> <remote-repo-url>`
So:
`git clone  --branch v0.0.1 https://github.com/nevermined-io/rapidsnark`